__tests__/app.test.js

```diff
@@ -48,4 +48,37 @@ describe('GET /api', () => {
             expect(body.endpoints).toEqual(endpoints)
         })
     })
-})
\ No newline at end of file
+})
+describe('GET /api/articles/:article_id', () => {
+    test.only('GET 200 sends an article object with the passed article_id with correct properties and values', () => {
```

Comment:
Please remove .only before pushing to github.
I cannot be sure any of your other tests work now this .only is here.